base/precompilation.jl

```diff
@@ -695,15 +692,19 @@ function precompilepkgs(pkgs::Vector{String}=String[];
         try
             wait(first_started)
             (isempty(pkg_queue) || interrupted_or_done.set) && return
-            fancyprint && lock(print_lock) do
-                printpkgstyle(io, :Precompiling, target)
-                print(io, ansi_disablecursor)
+            lock(print_lock) do
+                if target !== nothing
+                    printpkgstyle(io, :Precompiling, target)
+                end
+                if fancyprint
+                    print(io, ansi_disablecursor)
+                end
             end
             t = Timer(0; interval=1/10)
             anim_chars = ["◐","◓","◑","◒"]
             i = 1
             last_length = 0
-            bar = MiniProgressBar(; indent=2, header = "Progress", color = Base.info_color(), percentage=false, always_reprint=true)
+            bar = MiniProgressBar(; indent=0, header = "Precompiling packages ", color = :green, percentage=false, always_reprint=true)
```

Comment:
```suggestion
            bar = MiniProgressBar(; indent=0, header = "Precompiling packages", color = :green, percentage=false, always_reprint=true)
```
Otherwise there's an extra space before the bar
![Screenshot 2024-08-29 at 1 15 34 PM](https://github.com/user-attachments/assets/9f1a9c70-c26a-439b-83cc-2176b62e5c9c)

Reply:
Which we don't do for artifacts
![Screenshot 2024-08-29 at 1 19 44 PM](https://github.com/user-attachments/assets/6fdbad2b-bd18-4fdf-bc28-4e1149de75a3)

But we do have double space for registry downloads, which needs fixing in Pkg
![Screenshot 2024-08-29 at 1 21 42 PM](https://github.com/user-attachments/assets/627fadb2-b231-459e-83a5-2fede22beb95)

Reply:
It was to line up the start of the artifact and download bar with the precompile bar.

Reply:
Either way seems an improvement so let's go with it and see how it looks